assets/js/vue-apps/grant-detail.js

```diff
@@ -0,0 +1,42 @@
+import $ from 'jquery';
+import Vue from 'vue';
+
+function init() {
+    const mountEl = document.getElementById('js-grant-related-projects');
+    if (!mountEl) {
+        return;
+    }
+    new Vue({
+        el: mountEl,
+        delimiters: ['<%', '%>'],
+        data() {
+            return {
+                relatedGrants: false
+            };
+        },
+        mounted: function() {
+            $.ajax({
+                url: '/funding/search-past-grants-alpha/',
+                dataType: 'json',
+                data: {
```

Comment:
Clearly this is a massive `@TODO` – eventually this will be read out from a global object like with the main grant search. For now just getting something on the page was the goal.